src/plugins/presentation_panel/public/panel_component/panel_header/presentation_panel_hover_actions.tsx

```diff
@@ -0,0 +1,603 @@
+                {showDescription && (
+                  <EuiIconTip
+                    title={!hideTitle ? title || undefined : undefined}
+                    content={description}
+                    delay="regular"
+                    position="top"
+                    anchorClassName="embPanel__descriptionTooltipAnchor"
+                    data-test-subj="embeddablePanelDescriptionTooltip"
+                    type="iInCircle"
+                    css={css(`padding: $euiSizeXS;`)}
+                  />
+                )}
+                {quickActionElements.map(
+                  ({ iconType, 'data-test-subj': dataTestSubj, onClick, name }, i) => (
+                    <EuiToolTip key={`main_action_${dataTestSubj}_${api?.uuid}`} content={name}>
+                      <EuiButtonIcon
+                        iconType={iconType}
+                        color="text"
+                        onClick={onClick as MouseEventHandler}
+                        data-test-subj={dataTestSubj}
+                        aria-label={name as string}
+                      />
+                    </EuiToolTip>
+                  )
+                )}
+                {contextMenuActions.length ? (
+                  <EuiPopover
+                    repositionOnScroll
+                    panelPaddingSize="none"
+                    anchorPosition="downRight"
+                    button={ContextMenuButton}
+                    isOpen={isContextMenuOpen}
+                    className={contextMenuClasses}
+                    closePopover={() => setIsContextMenuOpen(false)}
+                    data-test-subj={
+                      isContextMenuOpen
+                        ? 'embeddablePanelContextMenuOpen'
+                        : 'embeddablePanelContextMenuClosed'
+                    }
+                    focusTrapProps={{
+                      closeOnMouseup: true,
+                      clickOutsideDisables: false,
+                      onClickOutside: (e) => {
+                        setIsContextMenuOpen(false);
+                      },
+                    }}
+                  >
+                    {menuPanelsLoading ? (
+                      <EuiContextMenuPanel
+                        className="embPanel__optionsMenuPopover-loading"
+                        title={i18n.translate('presentationPanel.contextMenu.loadingTitle', {
+                          defaultMessage: 'Options',
+                        })}
+                      >
+                        <EuiContextMenuItem>
+                          <EuiSkeletonText />
+                        </EuiContextMenuItem>
+                      </EuiContextMenuPanel>
+                    ) : (
+                      <EuiContextMenu
+                        data-test-subj="presentationPanelContextMenuItems"
+                        initialPanelId={'mainMenu'}
+                        panels={contextMenuPanels}
+                      />
+                    )}
```

Comment:
This is already contained within the `menuPanelsLoading` check above, right? So I don't think this ternary is necessary?

```suggestion
                    <EuiContextMenu
                      data-test-subj="presentationPanelContextMenuItems"
                      initialPanelId={'mainMenu'}
                      panels={contextMenuPanels}
                    />
```